products/compute/api.yaml

```diff
@@ -2404,6 +2421,13 @@ objects:
         description: |
           URL to a VpnTunnel that should handle matching packets.
         input: true
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'nextHopNetwork'
+        resource: 'Network'
+        imports: 'selfLink'
+        input: true
```

Comment:
Huh, we have this as computed in Terraform right now but the documentation definitely doesn't say it's output only (although https://cloud.google.com/vpc/docs/routes does, so we should probably also file a bug against whichever part of the documentation is wrong). Can you confirm for me that  this should be settable? If it is, it should probably get mentioned in the description above of types of nexthops that you can set